korio/src/commonTest/kotlin/com/soywiz/korio/file/std/UrlVfsTest.kt

```diff
@@ -1,10 +1,17 @@
 package com.soywiz.korio.file.std
 
+import com.soywiz.korio.async.suspendTest
+import com.soywiz.korio.stream.readAll
 import kotlin.test.*
 
 class UrlVfsTest {
     @Test
     fun test() {
         assertEquals("http://127.0.0.1:8080/", UrlVfs("http://127.0.0.1:8080")["."].absolutePath)
     }
+    @Test
+    fun testUrlVfsRead() = suspendTest {
+        assertTrue(UrlVfs("https://example.com").readAll().isNotEmpty())
+        assertTrue(UrlVfs("https://example.com").open().readAll().isNotEmpty())
```

Comment:
I don't like the idea on relying on external services for testing. This source set covers fast unit/integration tests but not end2end. We should use something mocked or open a temporal local server.

Let me check how to do mocked requests with korio 👀 and I will come back to you